client/public/locales/es/translation.json

```diff
@@ -36,38 +36,38 @@
-  "No Games loaded. Use <1>http://localhost:3000/#/g/local/FOLDER</1> to open a game directly from a local folder.": "No se ha cargado ningún juego. Use <1>http://localhost:3000/#/g/local/FOLDER</1> para abrir un juego directamente desde una carpeta local",
-  "<0>If you are considering writing your own game, you should use the <1>GameSkeleton Github Repo</1> as a template and read <3>How to Create a Game</3>.</0><1>You can directly load your games into the server and play it using the correct URL. The <1>instructions above</1> also explain the details for how to load your game to the server. We'd like to encourage you to contact us if you have any questions.</1><p>Featured games on this page are added manually. Please get in contact and we'll happily add yours.</p>": "<0>Si está considerando escribir su propio juego, use el <1>GameSkeleton Github Repo</1> como plantilla y lea <3>How to Create a Game</3>.</0><1>Puede cargar directamente los juegos en el servidor y jugarlo usando la URL adecuada. Las <1>instrucciones anteriores</1> también explican los detalles sobre cómo cargar su juego en el servidor. Le animamos a ponerse en contacto con nosotros si tiene preguntas.</1><p>Los juegos incluidos en esta página son añadidos manualmente. Por favor, contactenos y añadiremos el suyo encantados.</p>",
+  "A repository of learning games for the proof assistant <1>Lean</1> <i>(Lean 4)</i> and its mathematical library <5>mathlib</5>": "Un repositorio de juegos educativos sobre el asistente de demostración <1>Lean</1>, <i>(Lean 4)</i> y su librería matemática <5>mathlib</5> ",
+  "No Games loaded. Use <1>http://localhost:3000/#/g/local/FOLDER</1> to open a game directly from a local folder.": "No se ha cargado ningún juego. Usa <1>http://localhost:3000/#/g/local/FOLDER</1> para abrir un juego directamente desde una carpeta local",
+  "<0>If you are considering writing your own game, you should use the <1>GameSkeleton Github Repo</1> as a template and read <3>How to Create a Game</3>.</0><1>You can directly load your games into the server and play it using the correct URL. The <1>instructions above</1> also explain the details for how to load your game to the server. We'd like to encourage you to contact us if you have any questions.</1><p>Featured games on this page are added manually. Please get in contact and we'll happily add yours.</p>": "<0>Si quieres escribir tu propio juego, utiliza el repositorio de GitHub <1>GameSkeleton</1> como plantilla y lee el tutorial <3>Cómo Crear un Juego/3>.</0><1>Puedes cargar tus juegos directamente al servidor y jugarlos usando la URL adecuada. Las <1>instrucciones anteriores</1> también detallan cómo cargar tus juegos al servidor. Si tienes preguntas, puedes ponerte en contacto con nosotros.</1><p>Los juegos en esta página son añadidos manualmente. Por favor, contáctanos y añadiremos el tuyo con gusto.</p>",
```

Comment:
Shouldn't "How to Create a Game" be left untranslated? The [GameSkeleton](https://github.com/hhu-adam/GameSkeleton) repository doesn't seem to have a Spanish translation.

Reply:
Feels a bit weird to change language mid-sentence. Maybe I can rephrase as something like "see the guide on **how to make a game**" with the text then linking to the tutorial?